packages/api-proxy/src/v2/btc/dto/btc-transaction-output.dto.ts

```diff
@@ -19,6 +19,12 @@ export const EXAMPLE_LITECOIN_BTC_TRANSACTION_OUTPUT_DTO: BtcTransactionOutputDT
     address: "QS9mDWR42bcNK5CiWe2nU5PAM6vMzdi6fV",
   };
 
+export const EXAMPLE_BITCOIN_CASH_BTC_TRANSACTION_OUTPUT_DTO: BtcTransactionOutputDTO =
```

Comment:
```suggestion
export const EXAMPLE_BITCOIN_CASH_BTC_TRANSACTION_OUTPUT_DTO: BtcTransactionOutputDTO =
```
이거 타입 Btc 그대로 쓰면 안되는 애들도 있을듯?
DTO에 어노테이션으로 docs 내용이 들어가있다보니까.. 예를 들어 amount 같은 필드는 BTC 기준으로 적혀있을거라서 BCH 기준으로 새로 적으려면 dto 따로 만들어야할듯 ㅠ